docs/csharp/language-reference/attributes/pseudo-attributes.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Attributes interpreted by the compiler: Pseudo-attributes"
+ms.date: 01/2/2025
+description: "Learn about attributes you can add to code that are written to IL as modifiers. These custom attributes aren't emitted as attributes in the compiled output."
+---
+# Custom attributes that generate flags or options in the Intermediate Language (IL) output
+
+You add these attributes to your code for the compiler to emit specified Intermediate Language (IL) modifier. These attributes instruct the compiler to include the corresponding IL modifier in the output. These IL modifiers can be controlled in C# using these attributes:
+
+- <xref:System.Runtime.InteropServices.ComImportAttribute?displayProperty=fullName>: Specifies the `import` modifier.
```

Comment:
This might be nice as a table with the attribute in one column, the modifier in the second, and perhaps comments in a third column. Just a suggestion though.